packages/aragon-wrapper/src/configuration/index.js

```diff
@@ -0,0 +1,11 @@
+// Simple key-value configuration store
+const CONFIGURATION = {
+}
+
+export function getConfiguration (key) {
+  return key ? CONFIGURATION[key] : CONFIGURATION
```

Comment:
Should this a) check if the key exists? and return null if it does not?

Reply:
I think it's probably fine just to return `undefined` if the configuration doesn't exist 🤷‍♂

Reply:
ja! didn't catch that so yeah!